apps/scan/backend/src/app.ts

```diff
@@ -90,6 +91,15 @@ function buildApi(
       );
     },
 
+    generateLiveCheckQrCodeValue() {
+      const { machineId } = getMachineConfig();
+      const electionDefinition = workspace.store.getElectionDefinition();
+      return new LiveCheck().generateQrCodeValue({
```

Comment:
Will eventually dependency inject live check like I dependency inject the auth instance and artifact authenticator so that I can write proper app-level tests. Punted on this for now to keep this PR as minimal as possible